pkg/appmanager/appManager.go

```diff
@@ -906,7 +910,7 @@ func (appMgr *Manager) syncConfigMaps(
 		if cm.ObjectMeta.Namespace != sKey.Namespace {
 			continue
 		}
-		rsCfg, err := parseConfigMap(cm)
+		rsCfg, err := appMgr.parseConfigMap(cm)
```

Comment:
This may just be personal preference, but I think I would have rather passed the schemaLocal string to this function, rather than turning it into a class method.

Do you have a reason you went this route?

Reply:
I will change the according part of the code.

This was my first idea and since the function/method is only used by the AppMgr I thought it would be the best to make it a struct/class method